### PR TITLE
FIx to set correct gs:// path

### LINKED ISSUE #1121: OUTPUT_DIR with the gcs path does not work
`os.path.abspath()` removes one of the path slashes.

```
$ python
Python 3.6.9 (default, Jul  3 2019, 15:36:16)
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> path = 'gs://hoge/fuga/piyo/'
>>> OUTPUT_DIR = os.path.abspath(path)
>>> OUTPUT_DIR
'/home/User/workingdir/gs:/hoge/fuga/piyo'
```

https://github.com/blue-oil/blueoil/blob/r0.25/blueoil/environment.py#L42

### PR DESCRIPTION
**Pull request by iizukak**
## What this patch does to fix the issue.

Fix #1121

If `path` is started with `gs://`, then we do *not* add convert to local absolute path.

### CODE REVIEW

**Comment by iizukak** [comment]
I tested both `DATA_DIR` and `OUTPUT_DIR` are GCS path case manually.

File: blueoil/environment.py [hunk 1]
```diff
@@ -105,3 +105,13 @@ def teardown_test_environment():
...
+
+
+def is_gcs_path(path):
+    """Check argument string is GCS path or not
+    Args:
+        path: str
+    Returns:
+        True when string is GCS path
+    """
+    return path[0:5] == "gs://"
```
Comment (a-hanamoto) [inline blueoil/environment.py:117]:
```suggestion
    return path.startswith("gs://")
```

**Comment by iizukak** [comment]
OA

**Pull request merged**